### PR TITLE
SPORTSHUB-218 Added in the curved edges to the input for mobile search bar

### PR DESCRIPTION
**Pull request by ReggieStar11**
## Why was your change needed?

Please detail here why you did the change you did.

## What was Changed?

Please detail here what were the main changes you made.

## Any extra notes for reviewers to know?

Write any extra notes you may have for the PR.
This may include specific code paths that need extra scrutiny, or how to review a specific portion of code.

### CODE REVIEW

File: frontend/components/mobile/MobileSearchInput.tsx [hunk 1]
```diff
@@ -88,10 +80,10 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
           </button>
         </div>
         <div className="w-full flex items-center mt-7">
-          <MapPinIcon className="w-7 h-7 mr-2" />
+          <MapPinIcon className="w-7 h-7 mr-2 " />
```
Comment (41dan) [inline frontend/components/mobile/MobileSearchInput.tsx:83]: why is there a space after the 2 haha

File: frontend/components/mobile/MobileSearchInput.tsx [hunk 2]
```diff
@@ -101,22 +93,20 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
         </div>
         <div className="w-full mt-14 ml-1">
           <h3 className="font-semibold text-lg">Search by Tags</h3>
-          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll">
+          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll rounded-2xl">
             <TagGroup tags={tags} size="sm" />
           </div>
         </div>
         <div className="w-full mt-10 ml-1">
           <h3 className="font-semibold text-lg">Recent Searches</h3>
```
Comment (41dan) [inline frontend/components/mobile/MobileSearchInput.tsx:101]: image
Do we think showing recent searches in this way makes sense? I feel like the hyphen and location if there is no event name search seems a bit strange. Also tags is lowkey nonexistent how does it even show up?

**Review by 41dan — CHANGES_REQUESTED** [review]
yeah just fix up the space and we gucci

File: frontend/components/mobile/MobileSearchInput.tsx [hunk 3]
```diff
@@ -33,15 +33,15 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
...
+          : `h-screen bg-white w-screen absolute top-[2000px] left-0 z-50 transition-all duration-500 rounded-3xl`
       }
     >
       <div
         className={
           searchExpanded
-            ? `p-6 opacity-100 transition-all delay-500 duration-500`
-            : `p-6 opacity-0 transition-all`
+            ? `p-6 opacity-100 transition-all delay-500 duration-500 rounded-2xl`
+            : `p-6 opacity-0 transition-all rounded-2xl`
```
Comment (Edwin-Ni) [inline frontend/components/mobile/MobileSearchInput.tsx:44]: Why is this being rounded?

File: frontend/components/mobile/MobileSearchInput.tsx [hunk 4]
```diff
@@ -101,22 +93,20 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
         </div>
         <div className="w-full mt-14 ml-1">
           <h3 className="font-semibold text-lg">Search by Tags</h3>
-          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll">
+          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll rounded-2xl">
```
Comment (Edwin-Ni) [inline frontend/components/mobile/MobileSearchInput.tsx:96]: Why is this needed?

File: frontend/components/mobile/MobileSearchInput.tsx [hunk 5]
```diff
@@ -101,22 +93,20 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
...
-              <p className="text-sm font-light">
-                No recent searches... go ahead ;)
-              </p>
+              <p className="text-sm font-light">No recent searches... go ahead ;)</p>
             ) : (
               recentSearches.map((search, i) => {
                 const splitSearch = search.split(":");
                 return (
-                  <span key={i} className="flex items-center my-1">
+                  <span key={i} className="flex items-center my-1 rounded-lg">
```
Comment (Edwin-Ni) [inline frontend/components/mobile/MobileSearchInput.tsx:109]: What is the purpose?

**Review by Edwin-Ni — CHANGES_REQUESTED** [review]
A lot of unnecessary css?

File: frontend/components/mobile/MobileSearchInput.tsx (hunk 1 above)
Comment (ReggieStar11) [reply frontend/components/mobile/MobileSearchInput.tsx:83]: done

File: frontend/components/mobile/MobileSearchInput.tsx (hunk 2 above)
Comment (ReggieStar11) [reply frontend/components/mobile/MobileSearchInput.tsx:101]: i think this can be for later when mobile serach bar gets fixed

**Review by Edwin-Ni — APPROVED** [review]
lgtm
